routes/users.js

```diff
@@ -90,6 +91,12 @@ router.post(
   usersHandler.unblockUserV2
 );
 router.get('/check-follow', auth.isAuth, usersHandler.checkFollow);
+router.get(
+  '/check-follow-by-channelId',
+  auth.isAuth,
+  getTargetUserIdByPMChannelIdMiddleware,
+  usersHandler.checkFollow
+);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new route `/check-follow-by-channelId` is added correctly. It uses the `auth.isAuth` middleware for authentication and `getTargetUserIdByPMChannelIdMiddleware` to retrieve the target user ID. The final handler `usersHandler.checkFollow` is then used to check the follow status. Ensure that `usersHandler.checkFollow` is capable of handling the request when the target user ID is provided in the request query by `getTargetUserIdByPMChannelIdMiddleware`.
```diff
+ router.get(
+   '/check-follow-by-channelId',
+   auth.isAuth,
+   getTargetUserIdByPMChannelIdMiddleware,
+   usersHandler.checkFollow
+ );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->